src/main/scala/com/fulcrumgenomics/umi/FilterConsensusReads.scala

```diff
@@ -308,9 +308,9 @@ class FilterConsensusReads
       val bases   = rec.bases
       val quals   = rec.quals
       val depths1 = rec[Array[Short]](AbRawReadCount)
-      val depths2 = rec[Array[Short]](BaRawReadCount)
+      val depths2 = rec.get[Array[Short]](BaRawReadCount).getOrElse(depths1.map(_ => 0.toShort))
```

Comment:
This took some starting at for me to figure out.  It might be nice to add a note and maybe make it a little clearer (and more efficient) with:

```scala
// depth and errors may not be present if read was generated from a single strand
val (depths2, errors2) = {
  if (baMaxDepth > 0) (rec[Array[Short]](BaRawReadCount), rec[Array[Short]](BaRawReadErrors))
  else { val tmp = new Array[Short](bases.length); (tmp, tmp) }
}
```